modules/ROOT/pages/whats-new-in-studio.adoc

```diff
@@ -1,35 +1,31 @@
 = What’s New in Studio
 
-Anypoint Studio 7.13 introduces support for message flow visualization and management of DataWeave libraries through its UI.
+Anypoint Studio 7.14.0 introduces new features to enhance the users' experience making the daily use of Studio easier, faster, and more agile. 
 
-This version also improves the general user experience including new features such as the Exchange Comprehensive Publication.
+== Improved Copy-Paste Experience
 
-== Explore and Debug Shared Flows
+Now, users have the possibility to select if a suffix is added to the copy-pasted Mule Palette elements and to the flows to avoid having similar long names.
```

Comment:
to select if a suffix is added > to add a suffix

Reply:
Merging Diane's suggestion. Thanks!